### DISCUSSION TITLE
Not working on new empty project

### DISCUSSION THREAD

**Discussion by akrambak** [creativetimofficial/material-tailwind#833] [General]
I try to use @material-tailwind/react on new project, but it fail to compile

Typescipt

I followed this
https://www.material-tailwind.com/docs/react/installation

 ⨯ TypeError: (0 , _react.createContext) is not a function
    at [project]/node_modules/@material-tailwind/react/context/theme.js [app-rsc] (ecmascript) (........\material\.next\server\chunks\ssr\node_modules_@material-tailwind_react_351318._.js:13588:54)

**Comment by akrambak** [comment]
Use

"react": "18.2.0",
"react-dom": "18.2.0",

**Reply by carloshager** [reply]
I have the same problem of @akrambak, and forcing the react/react-dom version to 18.2.0 is not fixing the problem here as well
